src/Interpreter.test.js

```diff
@@ -22,10 +22,10 @@ function createInterpreter() {
 function createMockCommandHandler() {
     const mockCommandHandler = jest.fn();
     mockCommandHandler.mockImplementation(() => {
-        return new Promise((resolve, reject) => {
+        return new Promise((resolve) => {
             setTimeout(() => {
                 resolve();
-            }, 0);
+            }, 250);
```

Comment:
Do we need this change to the timeout value?

Reply:
IIRC that was left over from an early attempt to fix the race issues.  I reverted it.